include/alpaka/mem/buf/BufCpu.hpp

```diff
@@ -467,12 +467,28 @@ namespace alpaka
                                         cudaHostRegisterDefault),
                                     cudaErrorHostMemoryAlreadyRegistered);
 
+                                buf.m_spBufCpuImpl->m_bPinned = true;
+                            }
+#elif (defined(ALPAKA_ACC_GPU_HIP_ENABLED) && BOOST_LANG_HIP)
+                            if(buf.m_spBufCpuImpl->m_extentElements.prod() != 0)
+                            {
+                                // - cudaHostRegisterDefault:
+                                //   See http://cgi.cs.indiana.edu/~nhusted/dokuwiki/doku.php?id=programming:cudaperformance1
+                                // - cudaHostRegisterPortable:
+                                //   The memory returned by this call will be considered as pinned memory by all CUDA contexts, not just the one that performed the allocation.
```

Comment:
Do we really need to copy the CUDA specific documentation?

Reply:
no forgot to remove it.